app/worker.php

```diff
@@ -93,9 +93,14 @@
     return $register->get('statsd');
 }, ['register']);
 
+Server::setResource('pools', function ($register) {
+    return $register->get('pools');
+}, ['register']);
+
 $pools = $register->get('pools');
 $connection = $pools->get('queue')->pop()->getResource();
 $workerNumber = swoole_cpu_num() * intval(App::getEnv('_APP_WORKER_PER_CORE', 6));
+$workerNumber = 1;
```

Comment:
only one worker? I think we should remove this

Reply:
Removed